pkg/api/v1/conversion.go

```diff
@@ -262,6 +248,12 @@ func Convert_api_PodStatusResult_To_v1_PodStatusResult(in *api.PodStatusResult,
 		return err
 	}
 
+	if old := out.Annotations; old != nil {
```

Comment:
We copied annotation before (this change does not copy them, so any custom conversion that mutates annotations is responsible for copying them).